commit d02d77f334d8d5f627bba6082bacb24edf0a2c72
Author: Jimyeong Lee <jinmyeong.lee@linecorp.com>
Date:   Mon Jul 17 00:42:12 2023 +0900

    client: Add multi_target_id to handle libcephfs mds_spce=* command
    
    To fix multi target MDSs command result overwritten issue, add multi_target_id feature in CommandTable.
    Also, add multi_target_id in CommandOp to track all the multi target commands end to make formatted result
    
    Signed-off-by: Jimyeong Lee <jinmyeong.lee@linecorp.com>

diff --git a/src/mgr/MgrClient.h b/src/mgr/MgrClient.h
index a48ae163e18..1f9bb397fbe 100644
--- a/src/mgr/MgrClient.h
+++ b/src/mgr/MgrClient.h
@@ -54,6 +54,7 @@ class MgrCommand : public CommandOp
   bool tell = false;
 
   explicit MgrCommand(ceph_tid_t t) : CommandOp(t) {}
+  explicit MgrCommand(ceph_tid_t t, ceph_tid_t multi_id) : CommandOp(t, multi_id) {}
   MgrCommand() : CommandOp() {}
 };
 
